src/components/NotesMindMap/NotesMindMapSagas.js

```diff
@@ -63,6 +63,7 @@ function* changeSelectedNote({ data: { note, edges } }) {
 
 function* handleSearchResultClick({ data: { note } }) {
   const targetNote = note;
+  localStorage.setItem('lastViewedNoteId', targetNote.id);
```

Comment:
Why are we doing this only for the search result click, and not for a click on a note?
So if user did no search, there would be no last viewed note?
If user just starts with the root note and then selects another note, it should also be set as the last viewed note, because the user viewed it.

I think in `handleSearchResultClick` we can simply call `changeSelectedNote()`, and in `changeSelectedNote` we would set the `lastViewedNoteId`.